app/src/main/java/com/example/coen268/LoginActivity.java

```diff
@@ -207,13 +214,18 @@ public void onComplete(@NonNull Task<AuthResult> task) {
 
                             boolean isNew = task.getResult().getAdditionalUserInfo().isNewUser();
                             if (isNew) {
-                                // TODO fix account creation for Google authentication
+                                // TODO fix account creation for Google authentication - and sendBroadCast for FCM
                                 Intent intent = new Intent(LoginActivity.this, CreateAccountActivity.class);
                                 intent.putExtra(Constants.KEY_ACCOUNT_TYPE, accountType);
                                 intent.putExtra(Constants.KEY_IS_GOOGLE_AUTH, true);
                                 startActivity(intent);
                             } else {
                                 FirebaseUser user = mAuth.getCurrentUser();
+
+                                // Send logged in uid to fcm service
+                                Intent intent = new Intent(FCMService.FCM_CMDCHANNEL);
+                                intent.putExtra("uid", user.getUid());
```

Comment:
Does `intent.putExtra("Command", FCMService.CMD_LOGIN);` need to be here too? I noticed it was in the similar code above

Reply:
It doesn't need to be here; I put it here just for convenience. Could be cached to a instance var, but it's not necessary.